lib/galaxy/tools/deps/container_resolvers/mulled.py

```diff
@@ -333,10 +340,15 @@ def resolve(self, enabled_container_types, tool_info):
         if len(targets) == 0:
             return None
 
+        image_build = None
```

Comment:
I think the problem is https://github.com/galaxyproject/galaxy-lib/commit/3b8062d99200c1a3bb64ae0a2667acab75bda342#commitcomment-22516073 - sorry that I didn't spend more time looking into that. I'll redo this PR.